### ISSUE TITLE
Gitcoin grant contributions received api does not populate clr_round field

### ISSUE THREAD

**Issue by LefterisJP** [gitcoinco/web#9255]
Example call:

https://gitcoin.co/api/v0.1/grants/contributions_rec_report/?id=149&from_timestamp=2021-06-28&to_timestamp=2021-06-29&format=json

This is within clr_round 10. The `clr_round` field should be `10`. It's `null`.

**Comment by thelostone-mc** [comment]
Hmm now that you point it out ! Contributions can't be mapped to a clr_round 
Espically cause we might have a grant taking part in multiple rounds at the same time and we don't store at our end about which contribution happened for which round -> instead we'd be able to derive it

**Comment by thelostone-mc** [comment]
When a round is live, grant does story a link to that round and that's how the txnserializer ends up populating that info 
Once round is over, that link is removed (you could still derive this given the CLR round info )
TLDR: that field should have never been exposed in the first place

**Comment by LefterisJP** [comment]
Would you be able to derive it from the timestamp and still add it somehow? As this is actually quite useful information to know. If an event happened during a round or not.

**Comment by LefterisJP** [comment]
@thelostone-mc  ^
